src/ectrans4py/get_legendre_assets.F90

```diff
@@ -0,0 +1,70 @@
+SUBROUTINE GET_LEGENDRE_ASSETS(KRETURNCODE, KSIZEJ, KTRUNC, KSLOEN, KSPOLEGL, KLOEN, KNUMMAXRESOL, &
+  &                            KNMENG, PGW, PRPNM)
+! ** PURPOSE
+!    Simplified wrapper to TRANS_INQ for obtaining arrays necessary for performing Legendre transform
+!    (Gaussian weights, Legendre polynomials and NMENG (cutoff zonal wavenumber for each latitude))
+!
+! ** DUMMY ARGUMENTS
+!    KSIZEJ: number of latitudes in grid-point space
+!    KTRUNC: troncature
+!    KSLOEN: Size of KLOEN
+!    KSPOLEGL: Size of second dimension of Legendre polynomials
+!    KLOEN: number of points on each latitude row
+!    KNUMMAXRESOL: maximum number of troncatures handled
+!    KNMENG: cut-off zonal wavenumber
+!    PGW: Gaussian weights
+!    PRPNM: Legendre polynomials
+!
+! ** AUTHOR
+!    2 July 2025, S. Hatfield
+!
+! I. Dummy arguments declaration
+IMPLICIT NONE
+INTEGER(KIND=8), INTENT(OUT) :: KRETURNCODE
```

Comment:
Not suggesting the following for this PR, as the style here is done to match ectrans4py, but should we not aim to use kind specifiers that are more robust than `integer(kind=8)` ?
Something like `kind=int64` from the iso_fortran_env module might be an acceptable alternative?